returnn/tf/layers/rec.py

```diff
@@ -7857,7 +7863,12 @@ def sub_get_layer(sub_layer_name):
           _parent_layer_cache[sub_layer_name] = layer
       if not network.is_inside_rec_layer():
         # noinspection PyShadowingNames
-        source_data_ = layer.output.copy_template().copy_as_time_major()
+        mask_data = mask.output.copy_template()
+        masked_dim = mask_data.get_time_dim_tag()
+        if masked_dim not in layer.output.copy_template().dim_tags:
```

Comment:
Why the `copy_template`?

Reply:
Oh, I thought this was needed because there already was a line `source_data_.copy_template_replace_dim_tag`. So I assumed I need to use copy_template. Should I just leave that out and just write e.g. `mask.output`?